guides/doc-Release_Notes/master.adoc

```diff
@@ -2,7 +2,7 @@ include::common/attributes.adoc[]
 include::common/header.adoc[]
 
 ifndef::katello[]
-include::topics/foreman.adoc[]
+include::topics/katello-4.14.0.adoc[leveloffset=+1]
```

Comment:
Please insert this at the right place. There's an explicit instruction for this in https://community.theforeman.org/t/katello-4-14-rc1-release-procedure/39182 that does exactly that:
> `sed '/x.y.z releases here/a include::topics/katello-4.14.0.adoc[leveloffset=+1]' master.adoc`